server/src/user/dto/register.dto.ts

```diff
@@ -2,21 +2,19 @@ import { ApiProperty } from "@nestjs/swagger";
 
 export class RegisterRequestDto {
   @ApiProperty()
-  readonly id!: string;
+  readonly username!: string;
   @ApiProperty()
   readonly email!: string;
   @ApiProperty()
   readonly password!: string;
   @ApiProperty()
   readonly nickname!: string;
   @ApiProperty()
-  readonly studentIdNumber!: string;
-  @ApiProperty()
-  readonly realName!: string;
-  @ApiProperty()
   readonly isAdmin!: boolean;
   @ApiProperty()
   readonly isBanned!: boolean;
-  @ApiProperty({ enum: ["OB", "YB"] })
+  @ApiProperty({
+    enum: ["정보보호학과", "컴퓨터소프트웨어학과", "SecurityFirst"],
+  })
```

Comment:
Enum here is korean but it is english in user.schema. We need to remap it somewhere.